app/models/repair_params.rb

```diff
@@ -26,6 +26,14 @@ def diagnosed?
     sor_code.present?
   end
 
+  def contact_full_name
+    @answers.dig('contact_details', 'full_name')
+  end
+
+  def contact_telephone_number
+    @answers.dig('contact_details', 'telephone_number')
```

Comment:
Dig works well for optional data, because if keys are missing then it returns nil. 
Is that what we want here, or do we want to raise an exception if contact details are missing, like we do with `propertyReference`? Do we expect all repairs have this data mandatory?

Reply:
:+1: